extensions/objects/templates/filepath-editor.directive.ts

```diff
@@ -762,7 +770,40 @@ angular.module('oppia').directive('filepathEditor', [
               AlertsService.addWarning('Could not get resampled file.');
               return;
             }
-            ctrl.postImageToServer(dimensions, resampledFile);
+            ctrl.saveImage(dimensions, resampledFile);
+          }
+        };
+
+        ctrl.saveImageToLocalStorage = function(
```

Comment:
Do we need to flush the local storage once we upload the data? Also, assume someone deletes an image from RTE so will we remove that from local storage as well?

Reply:
Yeah, I haven't added upload yet, but yeah will flush it when uploading and deleting it.

Reply:
Done